rpcs3/Json/tooltips.json

```diff
@@ -37,6 +37,7 @@
 	"debug": {
 		"ppuDebug": "Never use this.",
 		"spuDebug": "Never use this.",
+		"enableTSX": "Force usage of TSX instructions on Haswell/Broadwell CPUs.\nOn older hardware this can lead to system instability, use it with caution.",
```

Comment:
In order to make this universal change the tooltip to something like:
"Force usage of TSX instructions.\nOn older Haswell or Broadwell CPUs this can lead to system instability, use it with caution."